src/features/todos.ts

```diff
@@ -1,8 +1,12 @@
-import { createSlice } from '@reduxjs/toolkit';
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
 import { Todo } from '../types/Todo';
 
-export const todosSlice = createSlice({
+export const { reducer, actions } = createSlice({
   name: 'todos',
   initialState: [] as Todo[],
-  reducers: {},
+  reducers: {
+    addTodos: (state, action: PayloadAction<Todo[]>) => {
+      state.push(...action.payload);
```

Comment:
The `addTodos` reducer uses `state.push(...action.payload)`, which mutates the state directly. While Redux Toolkit allows this due to its use of Immer, it's generally a good practice to return a new state array to avoid confusion. Consider using `return [...state, ...action.payload];` for clarity.